### ISSUE TITLE
Artisan Commands Fail: Add ability to configure PHP Path in ENV file.

### ISSUE THREAD

**Issue by bayareawebpro** [thedevdojo/voyager#2201]
- Laravel Version: 5.5.*
- Voyager Version: ^1.0
- PHP Version: 7.1.*
- Database Driver & Version: n/a

### Description:
Voyager needs the ability to define the php path in the env file.   Currently I can't use any of the artisan based features because both my local and remote environments use multiple versions of php.  For example if I want to use php 7.1, I must specify a handler which uses a binary named php71.    

If you could simply add the ability to define the php path in the env file that would help developers in my position.  Thank you!

### Steps To Reproduce:
Install on a server where php7 binary is not in expect path or uses unexpected file name.

**Comment by fletch3555** [comment]
This really isn't a Voyager issue, since default Laravel commands use the default php as well.  You can try something like this: https://stackoverflow.com/a/31839067

This is as far as I can go since this issue isn't specific to Voyager.

**Comment by bayareawebpro** [comment]
That's actually Incorrect.  Artisan commands can be run by specifying the php path in the command.  "any/path/I/want/php artisan something".  Laravel also allows us to change that in our composer file as well which is why I've had no issues up until now.

Your implementation of the Symfony process component assumes the php path when it can be specified easily with an ENV variable,  literally less then one line of code.  I would do it myself but I don't want to make a fork for one line of code.

Literally one function call to env() would solve the issue.  

OLD
`$process = new Process('cd '.base_path().' && php artisan '.$command.$args);`

NEW
`$process = new Process('cd '.base_path().' && '.env('PHP_PATH', 'php').' artisan '.$command.$args);`

**Comment by fletch3555** [comment]
yeah, just found it myself as well... hmm, okay.  I agree that it could be a problem, but I don't necessarily agree with your env solution.  I'll reopen this pending a solution.

**Comment by bayareawebpro** [comment]
Thanks for understanding.  I appreciate all the effort that when into the package. Amazing work.  Looking forward to having this as a replacement for my WP clients to get them onboard with the Laravel ecosystem :)
